cns/ipampoolmonitor/ipampoolmonitor.go

```diff
@@ -112,7 +117,23 @@ func (pm *CNSIPAMPoolMonitor) increasePoolSize() error {
 		return err
 	}
 
+	// Query the max ip count
+	maxIpCount:= pm.scalarUnits.MaxIPCount
```

Comment:
How will handle you existing clusters where NNC is not updated. @thatmattlong  ?

Reply:
For cases where max pods is not set it will unmarshal as 0, so @pjohnst5 let's take max to be 250 if `pm.scalarUnits.MaxIPCount == 0`

Reply:
Ok